openassessment/templates/openassessmentblock/edit/oa_edit.html

```diff
@@ -24,7 +24,7 @@ <h2 class="openassessment_alert_title">{% trans "Rubric Change Impacts Settings
                 <p class="openassessment_alert_message">{% trans "A change that you made to this assessment's rubric has an impact on some examples laid out in the settings tab. For more information, go to the Settings section and fix areas highlighted in red." %}</p>
             </div>
             <a href="" rel="view" class="action openassessment_alert_close">
-                <i class="icon-remove-sign"></i>
+                <i class="icon fa fa-times-circle" aria-hidden="true"></i>
```

Comment:
Note to reviewers: this icon missed the fa conversion. I verified that it does work with the new classes.